daemon/clients/investment/ipo.go

```diff
@@ -107,14 +108,15 @@ func (c *IPOClient) SendData() error {
 			UpdatedAt:       time.Now(),
 			Metadata:        "INVESTMENT_IPO_RECORD",
 			Content:         "",
-			Name:            target.Name,
+			Name:            fmt.Sprintf("%v (%v) goes public on %v", target.Name, target.Ticker, t),
```

Comment:
where is t defined?

Reply:
t, _ := time.Parse(layoutISO, availBefore)

Reply:
ok